.circleci/config.yml

```diff
@@ -95,16 +95,17 @@ commands:
           name: Building docker image for << parameters.target >>
           command: |
             build_tag="${CIRCLE_SHA1}"
-            [ "<< parameters.target >>" == "beta" ] && build_tag="beta-${CIRCLE_SHA1}"
-            docker build -t ${DOCKHUB_ORGANISATION}/binary-static-bot:${build_tag} -t ${DOCKHUB_ORGANISATION}/binary-static-bot:latest .
+            build_latest="latest"
+            [ "<< parameters.target >>" == "beta" ] && build_tag="beta-${CIRCLE_SHA1}" && build_latest="beta-latest"
```

Comment:
also `build_latest` should be a parameter that would either have a value "beta-latest" or "latest" from workflow. `build_tag` value can be either CIRCLE_SHA1 or CIRCLE_TAG which should be set from workflow